.drone.star

```diff
@@ -1899,3 +1908,40 @@ def getOcislatestCommitId(ctx):
             ],
         },
     ]
+
+def cacheChromium():
+    return [{
+        "name": "upload-chromium-cache",
+        "image": MINIO_MC,
+        "environment": minio_mc_environment,
+        "commands": [
+            "playwright_version=$(bash tests/drone/script.sh get_playwright_version)",
+            "mc alias set s3 $MC_HOST $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY",
+            "mc cp -r -a %s/.playwright s3/$CACHE_BUCKET/web/$playwright_version/" % dir["web"],
+            "mc ls --recursive s3/$CACHE_BUCKET/web",
+        ],
```

Comment:
we should zip the `.playwright` folder and push the archived file to the cache
See https://github.com/owncloud/ocis/blob/46d397d74ff8cda2197746fed3708eab9f34d13f/.drone.star#L2966-L2985